packages/container/src/components/FieldPluginContainer.tsx

```diff
@@ -37,30 +37,37 @@ import { ContentView } from './ContentView'
 import { StringParam, useQueryParam, withDefault } from 'use-query-params'
 import { ObjectView } from './ObjectView'
 import { UrlView } from './UrlView'
+import { StoryData } from '@storyblok/field-plugin'
 
-const uid = () => Math.random().toString(32).slice(2)
-
-const wrapperHost = 'localhost:7070'
 const defaultUrl = 'http://localhost:8080'
+const initialStory: StoryData = {
+  content: {},
+  lang: 'default',
+}
 const initialContent = ''
 const initialHeight = 300
 const initialWidth = 300
 
-const pluginParams: PluginUrlParams = {
-  uid: uid(),
-  host: wrapperHost,
-  secure: false,
-  preview: true,
-}
-
-type TestStory = { content: { count: number } }
-
 const UrlQueryParam = withDefault(StringParam, defaultUrl)
 
+const usePluginParams = () => {
```

Comment:
why useMemo instead of a const variable?

Reply:
It could be a const outside of the function

```
const pluginParams = {
  uid: Math.random().toString(32).slice(2),
  host: window.location.host,
  secure: window.location.protocol === 'https:',
  preview: true,
}

export const usePluginParams = () => {
  return pluginParams
}
```

But that would read from `window.location` when the module is being imported.

If it's inside the jook, then it's executed when the function is called, which I think is nicer. Then `useMemo` is needed to preserve referential equality, and to preserve the `uid`.

Reply:
Thanks for the explanation. It makes sense!